### DISCUSSION TITLE
【职场】 面试咨询+职业规划

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/home#223] [Ideas]
http://www.apachecn.org/map/ 

 ApacheCN 专注于优秀项目维护的开源组织

**Comment by jiangzhonglian** [comment]
梳理目标和制定学习计划，并辅助完成一个学习疗程，可以留言：

> 针对人群

你是不是: 学习迷茫，能力太差，学历低，还找不到工作？ 免费帮忙分析问题

> 服务指导

* 1. 学习方向 梳理
    * 进行个人评估，分析简历问题，给出学习建议
* 2. 简历指导+模版
    * 分享学习模版，分析简历问题，给出优化简历
* 3. 面试技巧分享
    * 分析面试遇到的问题，解答各种技巧
* 4. 模拟面试
    * 熟悉面试过程，提前适应面试流程

**Comment by vendanner** [comment]
## 咨询信息

1. 50元  支付宝
2. 597819425 +  2018-10-11 21:30 有空

## 学习计划 

> 用户: hhh

| 序号 | 完成时间 | 算法内容 | 完成进度 | 更新时间 | 
| --- | --- | --- | --- | --- |
| 1  | 10.15 – 10.21 | KNN,决策树，Bayesian   |  100%  |  20181021  |
| 2  | 10.22 – 10.28 | 逻辑回归，线性回归       |  100%  | 20181105    |
| 3  | 10.29 – 11.04 | 集成方法，树回归        |  100%  |    20181118|
| 4  | 11.05 – 11.11 | SVM                   |  100%  |  20181122  |
| 5  | 11.12 – 11.18 | K-Means 聚类           |  100%  |  20181202  |
| 6  | 11.19 – 11.25 | Apriori算法            |  100%  |  20181210  |
| 7  | 11.26 – 12.02 | FP-growth算法          |  100%  | 20181214   |
| 8  | 12.03 – 12.09 | PCA简化数据             |  100%  |20181203    |
| 9  | 12.10 – 12.16 | SVD简化数据             | 100%  |  20181208  |
| 10 | 12.17 – 12.23 | 大数据与MapReduce       |  100%  |   20181214 |
| 11 | 12.24 – 12.30 | 推荐系统                | 100%  | 20181219   |

## 学习总结

Github: https://github.com/vendanner/MLearn

**Comment by jianfengMan** [comment]
## 咨询信息

1. 50元 微信
2. 267994391 +  2018-10-09 22:00 有空

## 学习计划 

> 用户: 小布

| 序号 | 完成时间 | 算法内容 | 完成进度 | 更新时间 | 
| --- | --- | --- | --- | --- |
| 1  | 2018-10-21 | MlFoundation       | 100% | 20181012 |
| 2  | 2018-10-28 | KNN                | 100% | 20181015 |
| 3  | 2018-11-04 | DecisionTree      | 100% | 20181017 |
| 4  | 2018-11-11 | NaiveBayes         | 100% | 20181018 |
| 5  | 2018-11-18 | Logistic           |  100% | 20181023 |
| 6  | 2018-11-25 | SVM                |  100% | 20181028 |
| 7  | 2018-12-02 | AdaBoost           |  100% | 20181031|
| 8  | 2018-12-09 | RandomForest      |  100% | 20181103|
| 9  | 2018-12-16 | Regression         |  100% | 20181107|
| 10 | 2018-12-23 | RegTrees           |  100% | 20181119|
| 11 | 2018-12-30 | Kmeans             |  100% | 20181125|
| 12 | 2019-01-06 | Apriori            |  100% | 20181127|
| 13 | 2019-01-13 | FrequentPattemTree |  100% | 20181202|
| 14 | 2019-01-20 | PCA                |  100% | 20181204|
| 15 | 2019-01-27 | SVD                |  100% | 20181206|
| 16 | 2019-02-03 | BigData                |  100% | 20181206|
| 17 | 2019-02-10 | Recommender                 |  100% | 20181207|
## 学习总结

Github: https://github.com/jianfengMan/mysparkmllib

**Comment by jianfengMan** [comment]
## 咨询信息

1. 50元 微信
2. 267994391 +  2018-12-09
3. 后续的方向问题

## 职业方向 

> 用户: 小布

3年： Java 2年   大数据 1年
spark 工程方向深入研究，做spark行业比较了解算法（全面了解）

1. [leetcode 刷题](https://github.com/apachecn/awesome-algorithm)（简单 10  中级 7个  高级 3个）
2. spark 这方面你比我熟悉，自己准备一些（优化、性能和项目）
3. 项目正常准备，简单的过一下就行
4. 全面了解算法，实现的 [GitHub地址](https://github.com/jianfengMan/mysparkmllib) 发给别人。（重点找2~3个算法研究彻底就行 .. ）

**Comment by tabtou** [comment]
## 咨询信息

1. 50元 微信
2. 807507917 +  2018-12-28 
3. 职业方向问题

## 职业方向 

> 用户: tabtou

3年： 运维1年  大数据 1.5 年 算法 0.5年

1. 广告CTR预估： 
    [逻辑回归](https://github.com/apachecn/AiLearning/blob/dev/blog/ml/5.Logistic回归.md)、[GBDT](https://www.cnblogs.com/peizhe123/p/5086128.html)、[DNN](https://www.cnblogs.com/charlotte77/p/5629865.html)（深入去了解业界经典的算法，并且复现，了解业界前沿的技术）
2. 其他的算法来说，有空再了解（[分类、回归、聚类、降维](https://github.com/apachecn/AiLearning)）
3. 团队管理：职责定位和参与过程，选择好适合自己的环境展现自己。

**Comment by shitaokingstar** [comment]
## 咨询信息

1. 50元 微信
2. 377219613 +   2019-02-15 21:30 有空
3. 制定学习计划

## 学习计划 

CV 需要准备:
* [python基础](https://study.163.com/course/introduction.htm?courseId=1005859011#/courseDetail?tab=1)  需要了解：numpy 和 pandas <- 百度就行，哈哈
* [机器学习实战](https://github.com/apachecn/ailearning)
* [sklearn](http://sklearn.apachecn.org)、[spark](http://spark.apachecn.org)、[pytorch](http://pytorch.apachecn.org)、[tensorflow](https://tensorflow.google.cn)
* 项目: [kaggle](https://github.com/apachecn/kaggle)、天池
* 面试：[数据结构刷题](https://github.com/apachecn/awesome-algorithm)

> 用户: 晨曦

* 1个月：Python 基础 
 （网易云课程完成：02.15-02.19） 
    * 渠道：淘宝、网易云课堂
    * 重点：敲、使劲地敲、做笔记【1年后就知道为什么】
* 2～3个月：巩固机器学习
    * 听理论（不要纠结一步一步的推导）
    * 复线我们整理的代码【用Python 2.7，主要用于学习， 3.6不要尝试】

| 序号 | 完成时间 | 算法内容 | 完成进度 | 更新时间 | 
| --- | --- | --- | --- | --- |
| 1  | 2019-03-08 | MlFoundation       | 100% | 20190308 |
| 2  | 2019-03-11 | KNN                | 100% | 20190311 |
| 3  | 2019-03-13 | DecisionTree      | 100% | 20190313 |
| 4  | 2018-04-01 | NaiveBayes         | 100% | 20180401 |
| 5  | 2018-04-03 | Logistic           |  100% | 20180403 |
| 6  | 2019-03-11~22 | SVM                |  100% | 20190322 |
| 7  | 2019-04-06 | AdaBoost           |  100% | 20190406|
| 8  | 2019-04-08 | RandomForest      |  100% | 20190408|
| 9  | 2019-14-15 | Regression         |  100% | 20190415|
| 10 | 2018-12-23 | RegTrees           |  100% | 20181119|
| 11 | 2018-12-30 | Kmeans             |  100% | 20181125|
| 12 | 2019-01-06 | Apriori            |  100% | 20181127|
| 13 | 2019-01-13 | FrequentPattemTree |  100% | 20181202|
| 14 | 2019-01-20 | PCA                |  100% | 20181204|
| 15 | 2019-01-27 | SVD                |  100% | 20181206|
| 16 | 2019-02-03 | BigData                |  100% | 20181206|
| 17 | 2019-02-10 | Recommender                 |  100% | 20181207|

---

## 学习总结

Github:

**Comment by jiangzhonglian** [comment]
1
